lib/ffi/aspell/aspell.rb

```diff
@@ -13,8 +15,23 @@ module FFI
   # For more information see {FFI::Aspell::Speller}.
   #
   module Aspell
-    extend   FFI::Library
-    ffi_lib ['aspell', 'libaspell.so.15']
+    extend FFI::Library
+
+    # Allow custom pathing to library path
+    if !!ENV['ASPELL_LIB_PATH']
+      ffi_lib ENV['ASPELL_LIB_PATH']
+    else
+      case ::RbConfig::CONFIG['host_os']
+      when /linux/
+        ffi_lib ['aspell', 'libaspell.so.15']
+      when /darwin/
+        if File.exist?('/opt/boxen/homebrew/lib/libaspell.dylib')
```

Comment:
I think it's better to have a separate approach instead of using a `case` here to add all kinds of application specific code. We now have Boxen _and_ Homebrew, and apparently neither know how to properly add libraries to the search path. 

First, did you run `brew link aspell`? If I remember correctly that _should_ ensure the library can be found: https://github.com/Homebrew/brew/blob/master/docs/FAQ.md#can-i-install-my-own-stuff-to-usrlocal

Second, does adding `libaspell` to `ffi_lib ['aspell', 'libaspell.so.15']` not solve the problem? Looking at the filenames below that _might_ ensure the library gets picked up.

Either way, I _really_ don't want platform specific logic like this; let alone _package manager_ specific logic. It becomes a big mess way too quickly and it's a pain to maintain.